src/lib/list/selection-list.ts

```diff
@@ -392,16 +397,15 @@ export class MatSelectionList extends _MatSelectionListMixinBase implements Focu
 
   /** Passes relevant key presses to our key manager. */
   _keydown(event: KeyboardEvent) {
-    if (this.disabled) {
-      return;
-    }
-
     switch (event.keyCode) {
       case SPACE:
       case ENTER:
-        this._toggleSelectOnFocusedOption();
         // Always prevent space from scrolling the page since the list has focus
         event.preventDefault();
+
+        if (!this.disabled) {
+          this._toggleSelectOnFocusedOption();
```

Comment:
I'd move the `preventDefault` into the check as well. That's what it used to do before.